packages/neuron-wallet/package.json

```diff
@@ -12,7 +12,7 @@
   "main": "dist/main.js",
   "license": "MIT",
   "scripts": {
-    "start": "tsc && electron .",
+    "start": "tsc && cross-env NODE_ENV=development electron .",
```

Comment:
Please don't alter start/run task scripts with other feature/updates. It has cause too much issue and is hard to revert to trouble shooting.